### PR TITLE
policies: rename list fields to be plural

### PR DESCRIPTION
**Pull request by josedonizetti**
### 1. Explain what the PR does

Fix https://github.com/aquasecurity/tracee/issues/3241
Fix https://github.com/aquasecurity/tracee/issues/3181

### 2. Explain how to test it

This PR does two things:

1 - renames a few fields of the policy, and fix the code where appropriate.
     filter -> filters
     action -> actions

2 - add support to multiple default actions
      defaultAction -> defaultActions

### 3. Other comments

This PR will break the internal tests, because of the policies change.

### CODE REVIEW

File: deploy/helm/tracee/templates/tracee-policies.yaml [hunk 1]
```diff
@@ -9,8 +9,9 @@ data:
   signatures.yaml: |-
     name: signature_events
     description: traces all signature events
-    defaultAction: log
-    scope:
+    defaultActions: 
+      - log
+    scopes:
```
Comment (yanivagman) [inline deploy/helm/tracee/templates/tracee-policies.yaml:14]: This should be kept as scope and not scopes (we have one scope, which is defined by several filters)

Comment (josedonizetti) [reply deploy/helm/tracee/templates/tracee-policies.yaml:14]: ok

**Comment by yanivagman** [comment]
Still a draft?

**Review by rafaeldtinoco — APPROVED** [review]
LGTM

**Pull request merged**